unified_planning/io/pddl_writer.py

```diff
@@ -181,11 +181,10 @@ def __init__(
     ):
         walkers.DagWalker.__init__(self)
         self.get_mangled_name = get_mangled_name
-        self.simplifier = env.simplifier
 
     def convert(self, expression):
         """Converts the given expression to a PDDL string."""
-        return self.walk(self.simplifier.simplify(expression))
+        return self.walk(expression.simplify())
```

Comment:
Why this change? In this way you are forcing the use of the "global" simplifier instead of the one in the environment used to construct this converter, no?